editor/editor_node.cpp

```diff
@@ -2161,9 +2161,14 @@ void EditorNode::_edit_current() {
 	if (!inspector_only) {
 		EditorPlugin *main_plugin = editor_data.get_editor(current_obj);
 
-		for (int i = 0; i < editor_table.size(); i++) {
-			if (editor_table[i] == main_plugin && !main_editor_buttons[i]->is_visible()) {
-				main_plugin = nullptr; //if button is not visible, then no plugin active
+		int p_which = 0;
```

Comment:
We usually use the `p_` prefix for method parameters, so it's misleading to use that convention for a local variable. Maybe `plugin_index` would be better?

Reply:
updated